index.html

```diff
@@ -339,6 +339,21 @@ <h3>Success Criteria</h3>
             </li>
           </ul>
         </div>
+        <div>
+          <h3>Interoperability</h3>
+              <p>The working group will define a syntax for DIDs, a data model for DID Documents, and the requirements for DID methods. Interoperability of implementations for DIDs and DID Documents will be tested by evaluating an implementation's ability to create and parse DIDs and DID Documents that conform to the specification. Interoperability for DID methods will be determined by evaluating each DID method's specification to 
+              determine, at a minimum, </p>
+              <ol>
+                  <li> the required operations are supported,</li>
```

Comment:
<li> the DID method name is unique and not used by an existing, incompatible DID method, </li>